paymentsheet/src/test/java/com/stripe/android/paymentsheet/viewmodels/PrimaryButtonUiStateMapperTest.kt

```diff
@@ -32,11 +32,12 @@ class PrimaryButtonUiStateMapperTest {
 
         val mapper = createMapper(
```

Comment:
can you add some tests for the behavior changes you made in PrimaryButtonUiStateMapper?